FactoryMethod/Projects/Logistics/src/Destination.h

```diff
@@ -0,0 +1,33 @@
+//
+// Created by @PolinaShlepakova on 16.02.2019.
+//
+
+#ifndef LOGISTICS_DESTINATION_H
+#define LOGISTICS_DESTINATION_H
+
+#include <string>
+#include <iostream>
+
+class Destination {
+public:
+	Destination(const std::string& country, const std::string& city,
+				const std::string& addressLine, const std::string& index = "");
```

Comment:
@PolinaShlepakova, why does `index` able to be empty string by default?

Reply:
Because not all countries have it.